scripts/setup-passcode.sh

```diff
@@ -10,12 +10,10 @@ if [[ -z $OKTA_DOMAIN ]]; then
     exit 1
 fi
 
-oktamate=./scripts/oktamate
-
 # Delete user if it exists to not fail the script
-existing_user=$($oktamate users -k $OKTA_API_KEY -d $OKTA_DOMAIN | grep $PASSCODE_USERNAME)
+existing_user=$(oktamate users -k $OKTA_API_KEY -d $OKTA_DOMAIN | grep $PASSCODE_USERNAME)
 if [[ -n $existing_user ]]; then
-    $oktamate users delete -k $OKTA_API_KEY -d $OKTA_DOMAIN $PASSCODE_USERNAME
+    oktamate users delete -k $OKTA_API_KEY -d $OKTA_DOMAIN $PASSCODE_USERNAME
 fi
 
-$oktamate users create -k $OKTA_API_KEY -d $OKTA_DOMAIN -p $PASSCODE_PASSWORD -g '00gua62f9B5eLhVBx5d6' $PASSCODE_USERNAME
+oktamate users create -k $OKTA_API_KEY -d $OKTA_DOMAIN -p $PASSCODE_PASSWORD -g 'Passcode' $PASSCODE_USERNAME
```

Comment:
Please remove the API key and okta domain arguments from these calls, otherwise the API key may leak into the CI console logs inadvertently. Just rely on the environment variables.